regression/strings/StringContains01/test.java

```diff
@@ -0,0 +1,8 @@
+public class test
+{
+    public static void main(String s)
+    {
+        String ab = "abc";
+	assert(ab.contains(s));
```

Comment:
Weird - looks very different in the "Files" view rather than this view - maybe a mixture of tabs and spaces?